advanced/helm/pattern-1/README.md

```diff
@@ -45,25 +47,34 @@ Git repository. <br>
 git clone https://github.com/wso2/kubernetes-apim.git
 ```
 
-##### 2. Setup a Network File System (NFS) to be used for persistent storage.
+##### 2. Setup persistent storage.
 
-Create and export unique directories within the NFS server instance for each of the following Kubernetes Persistent Volume
-resources defined in the `<HELM_HOME>/apim-with-analytics-conf/values.yaml` file:
+* Using Azurefiles
+  
+  Add the following parameter and value to the values.yaml.
+  ```
+  cloudProvider: Azure
+  ```
+  
+* Using a Network File System (NFS)
 
-* `sharedDeploymentLocationPath`
-* `analyticsDataLocationPath`
-* `analyticsLocationPath`
+  Create and export unique directories within the NFS server instance for each of the following Kubernetes Persistent Volume
+  resources defined in the `<HELM_HOME>/apim-with-analytics-conf/values.yaml` file:
 
-Grant ownership to `wso2carbon` user and `wso2` group, for each of the previously created directories.
+  * `sharedDeploymentLocationPath`
```

Comment:
The Persistent Volume configurations that are parameterized differ when compared with what is there in the README [documentation](https://github.com/wso2/kubernetes-apim/blob/0d95d0a3df2816f491b57adb09e8b4a47e3855f2/advanced/helm/pattern-1/apim-with-analytics/values.yaml#L8-L9).

Reply:
Changed to,
* sharedDeploymentLocationPath
* sharedTenantsLocationPath